bythepixel/kyrio-go2broadband
# 19-check-box-misalignment

Closes #19 

## How should this be tested?
- Navigate to the homepage
- Toggle the **I would like to receive special offers** checkbox
- Resize the window to below 1000px wide
- See checkbox 'active' toggle pseudo element contained within the checkbox
